src/es/components/atoms/scrollToTop/ScrollToTop.js

```diff
@@ -0,0 +1,90 @@
+// @ts-check
+import { Shadow } from '../../prototypes/Shadow.js'
+
+/**
+* @export
+* @class ScrollToTop
+* @type {CustomElementConstructor}
+*/
+export default class ScrollToTop extends Shadow() {
+  constructor(options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+     this.button = this.root.children[0];
+
+  }
+
+  connectedCallback() {
+    if (this.shouldRenderCSS()) this.renderCSS()
+    this.button.addEventListener('click', () => {
```

Comment:
this.clickEventListener = event =>... define in constructor, so that you can remove the event listener in the disconnected callback